devlab/tests/testcases/test_verify_dst_functionality.py

```diff
@@ -23,10 +23,12 @@
 
 import devlab.tests.config as config
 import devlab.tests.functional_test as functional_test
+from devlab.tests.base import BasePrerequisites as base
```

Comment:
Ok, i think we'll move this comment to a different task. I will organize all tests imports when will do a package for it.